### PR TITLE
fix(Tabs): fix error with a single Tabs.Content as children

### PR DESCRIPTION
**Pull request by langz**
This should:
- Fix the errors when providing Tabs component with a single Tabs.Content as `children`
- Adds screenshot tests and unit tests for when `data` prop has a single element and when Tabs component has a single Tabs.content as `children`.

Here's how it looked and behaved when providing Tabs component with a single Tabs.Content as children:
Screenshot 2023-07-27 at 14 56 50
Screenshot 2023-07-27 at 14 57 45
Screenshot 2023-07-27 at 14 57 18

And when pressing the Tabs title you would get:
Screenshot 2023-07-27 at 14 57 59

Now after the fix, the same code will not result in an error, and it will look like this:

image

### CODE REVIEW

**Review by tujoworker — APPROVED** [review]
Looks good 👏

File: packages/dnb-eufemia/src/components/tabs/__tests__/Tabs.screenshot.test.ts [hunk 1]
```diff
@@ -21,6 +21,22 @@ describe('Tabs', () => {
...
 
+  it('have to match single single children as react element', async () => {
+    const screenshot = await makeScreenshot({
+      selector:
+        '[data-visual-test="tabs-single-children-react-element"] .dnb-tabs__tabs',
+    })
+    expect(screenshot).toMatchImageSnapshot()
+  })
+
+  it('have to match single single element in data', async () => {
```
Comment (langz) [inline packages/dnb-eufemia/src/components/tabs/__tests__/Tabs.screenshot.test.ts:32]:
```suggestion
  it('have to match single element in data', async () => {
```

File: packages/dnb-eufemia/src/components/tabs/__tests__/Tabs.screenshot.test.ts [hunk 2]
```diff
@@ -21,6 +21,22 @@ describe('Tabs', () => {
     expect(screenshot).toMatchImageSnapshot()
   })
 
+  it('have to match single single children as react element', async () => {
```
Comment (langz) [inline packages/dnb-eufemia/src/components/tabs/__tests__/Tabs.screenshot.test.ts:24]:
```suggestion
  it('have to match when used with a single children as react element', async () => {
```

File: packages/dnb-eufemia/src/components/tabs/Tabs.js [hunk 3]
```diff
@@ -199,56 +199,71 @@ export default class Tabs extends React.PureComponent {
...
+    }
+
     let res = []
 
     // check if we have to use the children prop to prepare our data
     const data =
       !props.data && props.children ? props.children : props.data
 
-    // if it is a React Component - collect data from Tabs.Content component
+    // if it is a array of React Components - collect data from Tabs.Content component
```
Comment (langz) [inline packages/dnb-eufemia/src/components/tabs/Tabs.js:246]:
```suggestion
    // if it is an array of React Components - collect data from Tabs.Content component
```

**Pull request merged**